src/wms/src/test/java/org/geoserver/wms/capabilities/GetCapabilitiesReponseTest.java

```diff
@@ -207,4 +207,29 @@ public void testAcceptLanguagesParameter() throws Exception {
                 "http://localhost:8080/geoserver/wms?request=GetLegendGraphic&format=image%2Fpng&width=20&height=20&layer=nature&Language=it",
                 "//Layer[Name='nature']/Style/LegendURL/OnlineResource/@xlink:href", dom);
     }
+
+    @Test
+    public void testNullLocale() throws Exception {
```

Comment:
What is this testing? that the backlinks do not contain language if the locale has not been provided?

Reply:
fixed.